src/components/todo-list.js

```diff
@@ -10,21 +11,36 @@ export default class TodoListComponent {
   mount() {
     this.mountPoint.innerHTML = this.render();
     this.querySelectors();
+    this.doneTask();
   }
 
   addTask(task) {
     this.list.innerHTML += `
-            <li class="todo-list__item">${task}</li>
+    <tr><td><input class ="checkbox" type="checkbox"></td>
+    <td>${task}</td>
+    <td><button value="&#215" style="width:5px;height:15px;"></button></td></tr>
         `;
   }
 
+  doneTask() {
+    let checkboxElements = document.getElementsByClassName("checkbox");
+
+    for (let i = 0; i < checkboxElements.length; i++) {
+      checkboxElements[i].addEventListener("change", function() {
+        if (this.checked) {
+          this.closest("tr").style.textDecoration = "line-through";
```

Comment:
Firstly, closest("tr") should be called on html element, "this" in your case is component. Call this method on checkboxElements[i]

Reply:
thanks! really doneTask() was getting called one time, that's why the style was not updating. 
problem has solved by calling doneTask() inside addTask()